src/features/navigation/RootNavigator/routes.ts

```diff
@@ -140,6 +141,15 @@ export const routes: Route[] = [
     path: 'creation-compte/confirmation',
     options: { title: t`Compte créé !` },
   },
+  {
+    name: 'AfterChangeEmailValidationBuffer',
+    component: AfterChangeEmailValidationBuffer,
+    pathConfig: {
+      path: 'changement-email',
+      deeplinkPaths: ['profil/modification-email/confirmation'],
```

Comment:
Vu que tu créer la page, tu n'as pas de support legacy à avoir.
Je propose 
```suggestion
      path: 'profil/modification-email/validation',
```

Reply:
En fait le deeplink a `changement-email`

En testant, les 2 qui marchent c'est :
```
      path: 'changement-email',
      deeplinkPaths: ['profil/modification-email/confirmation']
```
et 
```
      deeplinkPaths: ['changement-email'],
```

Reply:
y a déjà un lien utilisé en prod ou pas ?
si non fais ce que j'ai proposé